baseline.gradle

```diff
@@ -74,8 +65,8 @@ subprojects {
     spotless {
       java {
         target 'src/main/java/**/*.java', 'src/test/java/**/*.java', 'src/testFixtures/java/**/*.java', 'src/jmh/java/**/*.java', 'src/integration/java/**/*.java'
-        // we use an older version of google-java-format that is compatible with JDK 8
-        googleJavaFormat("1.7")
+        // we stick to a version of google-java-format that produces same result for JDK 11/17/21
```

Comment:
@manuzhang can you please link the issue from GJF that causes issues with 1.23.0. That way we can later check and update GJF whenever the issue is fixed